### PR TITLE
Added sequencify package

### PR DESCRIPTION
**Pull request by npenin**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- X ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Review by plantain-00 — APPROVED** [review]

File: types/sequencify/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Type definitions for sequencify 0.0
```
Comment (RyanCavanaugh) [inline types/sequencify/index.d.ts]: I can't guarantee that the `@types` publishing script won't choke on this. Can you change to `0.1`?

File: types/sequencify/sequencify-tests.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* Add tests for your definition file here */
+
+/// <reference types="node" />
+
+import * as sequencify from 'sequencify';
```
Comment (RyanCavanaugh) [inline types/sequencify/sequencify-tests.ts]: Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/sequencify/sequencify-tests.ts (hunk 2 above)
Comment (npenin) [reply types/sequencify/sequencify-tests.ts]: Even though it is not ES6 compliant, it is however working and compiling in typescript. As mentioned in the answer, it is so 2013, but the module is done that way. I cannot change the definition. I can update the code to `import sequencify = require('sequencify');`, but either way is working.

Comment (RyanCavanaugh) [reply types/sequencify/sequencify-tests.ts]: Please use the `require` form then

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Pull request merged**